### PR TITLE
qpwgraph: init at 0.0.9

### PR DESCRIPTION
**Pull request by kanashimia**
###### Motivation for this change
Added a tool similar to qjackctl, but that is native for pipewire.
It seems to work better than helvum even tho version number is that low :')  
It can also be compiled with Qt6, but nixpkgs doesn't have Qt6 at this point.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/qpwgraph/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{ lib, mkDerivation, fetchFromGitLab
+, cmake, pkg-config
+, alsa-lib, pipewire
+, ...
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/qpwgraph/default.nix:4]:
```suggestion
```

File: pkgs/applications/audio/qpwgraph/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}:
+
+mkDerivation rec {
+  pname = "qpwgraph";
+  version = "0.0.6";
+
+  src = fetchFromGitLab {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/qpwgraph/default.nix:11]: Great to see that you are using the gitlab fetcher.

File: pkgs/applications/audio/qpwgraph/default.nix (hunk 1 above)
Comment (kanashimia) [reply pkgs/applications/audio/qpwgraph/default.nix]: Thanks for catching this! Forgot that `callPackage` works like that.

File: pkgs/applications/audio/qpwgraph/default.nix (hunk 2 above)
Comment (kanashimia) [reply pkgs/applications/audio/qpwgraph/default.nix:10]: With recent github problems this sounds very ironic :^)

**Pull request merged**